web/models.py

```
@@ -2216,6 +2222,29 @@
+    token_id = models.CharField(max_length=100, blank=True)
+    contract_address = models.CharField(
+        max_length=100,
+        blank=True,
+    )
+    transaction_hash = models.CharField(max_length=100, blank=True)
+    metadata_uri = models.URLField(blank=True)
+    minted_at = models.DateTimeField(blank=True)
+    wallet_address = models.CharField(max_length=100, blank=True)
+    icon_url = models.URLField(blank=True)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)